src/main/java/nurseybook/commons/core/Messages.java

```diff
@@ -18,8 +18,8 @@
     public static final String MESSAGE_DUPLICATE_TASK = "This task already exists in the nursey book. \n"
             + "Hint: If you want to change the recurrence of a task, edit that specific task directly.";
     public static final String MESSAGE_NO_CHANGES = "Fields entered are the same as before.";
-    public static final String MESSAGE_NO_SUCH_ELDERLY = "Some of the elderly names are not found in your elderly "
-            + "database.\n"
+    public static final String MESSAGE_NO_SUCH_ELDERLY = "The elderly name you have entered is not found in your "
```

Comment:
I'm actually the one who created this variable. The reason is that there might be multiple names entered by the user that are invalid. Maybe it would be better if you change it to "Some of the elderly names that you have entered are not found..."

Reply:
Sorry! I was unaware that we can enter multiple elderly names when creating a task. Will change it back. 👍